src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -99,11 +99,28 @@ public function getInitialTestRunCommandLine(
     /**
      * Returns array of arguments to pass them into the Mutant Symfony Process
      *
+     * @param CoverageLineData[] $coverageTests
+     *
      * @return string[]
      */
-    public function getMutantCommandLine(MutantInterface $mutant, string $extraOptions): array
-    {
-        return $this->getCommandLine($this->buildMutationConfigFile($mutant), $extraOptions, [], false);
+    public function getMutantCommandLine(
+        array $coverageTests,
```

Comment:
Passing arrays around might need be worked around later.

Reply:
Could you please elaborate on that? this is an array of `CoverageLineData`, not just dumb native arrays

Reply:
Yes, yet it can be copied on write still. It is not as bad as an array of arrays. In a perfect world we never need to pass arrays as arguments, but you can do only as much. So that I'm not to say there's anything to be done right now, this also could be overengineering and/or premature optimization.

Reply:
Got it, thanks